### ISSUE TITLE
Error when cloning Core repo

### ISSUE THREAD

**Issue by Tom-LeFx** [Innoactive/Creator#252]
### Describe the bug
When trying to create a local branch to track master on this repo, i get lfs smudge errors for .documentation-contents

### Steps to Reproduce

1. Create local repo
2. add Innoactive-Creator-Core repo as remote
3. create new branch, set it to track Innoactive-Creatore/master
-->Error downloading object: .Documentation/articles/images/conditions/grabbable_properties.png (12f7c59): Smudge error: Error downloading .Documentation/articles/images/conditions/grabbable_properties.png (12f7c59de970c06649a1b9de30efbba5672bd7e7454dd8a878b8e5a11e28f6bc): [12f7c59de970c06649a1b9de30efbba5672bd7e7454dd8a878b8e5a11e28f6bc] Object does not exist on the server: [404] Object does not exist on the server

### Expected behavior
Clean, error-free tracking is possible

### Context (Environment)
According to research, the problem results from faulty pushed lfs content to the aforementioned folder:
https://stackoverflow.com/questions/46521122/smudge-error-error-downloading

### Possible Solution
"This corruption can happen due to another developer's misconfigured git-lfs that pushed to the repository, or sometimes due to some unusual failure during a push. Try re-sending the missing object (using its object ID) from a working tree:

git lfs push --object-id <remote> <oid>
The OID should be in the Smudge error message."

**Comment by SimonTheSourcerer** [comment]
Hallo Tom!
Tried to reproduce it but could not. What I did:

```
git init
git remote add origin git@github.com:Innoactive/Creator.git
git fetch --all
git checkout --track origin/master
```
Everything works as intended, no errors, the faulty image is there, exists and can be viewed.

I'm definitely not a git guru, did I do anything different than you?

**Comment by Tom-LeFx** [comment]
I agree - the initial checkout works. But try to checkout the commit marked with "2.7.0" in the master branch.
I get authentication-errors 

```
Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:Innoactive/Hub-Documentation-DocFx-Theme.git' into submodule path 'C:/UnityProjects/AUDI_DevStack/InnoactiveCreatorFork/vrt-innoactive-creator-test/.Documentation/theme' failed
Failed to clone '.Documentation/theme' a second time, aborting
```

The smudge error does not show up again in a new cloned repo - i agree.

**Comment by SimonTheSourcerer** [comment]
Yes, this Repo is still private, but only used for building the documentation. Is this breaking something for you or is it just an ugly message in the console? We could decouple both repositories, but this wont help for older tags :/
